ci-operator/step-registry/assisted/ofcir/setup/assisted-ofcir-setup-commands.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ assisted ofcir setup command ************"
+
+PACKET_CONF="$SHARED_DIR/packet-conf.sh"
+CIRFILE="$SHARED_DIR/cir"
+SSH_KEY_FILE="$CLUSTER_PROFILE_DIR/packet-ssh-key"
+ANSIBLE_CONFIG_FILE="${SHARED_DIR}/ansible.cfg"
+
+if [[ ! -f "$CIRFILE" ]]; then
+    echo "Error: CIR file not found at $CIRFILE"
+    exit 1
+fi
+
+if [[ ! -f "$PACKET_CONF" ]]; then
+    echo "Error: packet-conf.sh not found at $PACKET_CONF"
+    exit 1
+fi
+
+if [[ ! -f "$SSH_KEY_FILE" ]]; then
+    echo "Error: SSH key file not found at $SSH_KEY_FILE"
+    exit 1
+fi
+
+echo "executing packet-conf.sh..."
+# shellcheck disable=SC1090
+source "$PACKET_CONF"
+
+export SSH_KEY_FILE="$SSH_KEY_FILE"
+
+if [[ -z "${IP:-}" ]]; then
+    IP=$(jq -r '.ip' "$CIRFILE")
+fi
+export IP
+
+mkdir -p build/ansible
+cd build/ansible
+
+cat > prepare-ansible.yaml <<-EOF
```

Comment:
if we want to start clean with ofcir, I think we should externalize all our playbooks in a repo, and build an image, and use it to run these step. That way we decouple the CI from our scripts and it makes them re-usable on our laptop to test them.

Reply:
It would be great, but then we are losing the ability to test them with rehearsal

Reply:
you will test them on the repo that defines the playbooks. I think it's an ok trade off compared to what we have today.

Reply:
As discussed, this is not a blocker to merge this PR, it can be done later